sdk/storage/azure-storage-queue/tests/queuetestcase.py

```diff
@@ -233,53 +119,6 @@ def _create_storage_service(self, service_class, settings):
         self._set_test_proxy(service, settings)
         return service
 
-    # for blob storage account
-    def _create_storage_service_for_blob_storage_account(self, service_class, settings):
-        if hasattr(settings, 'BLOB_CONNECTION_STRING') and settings.BLOB_CONNECTION_STRING != "":
-            service = service_class(connection_string=settings.BLOB_CONNECTION_STRING)
-        elif settings.IS_EMULATED:
-            service = service_class(is_emulated=True)
-        elif hasattr(settings, 'BLOB_STORAGE_ACCOUNT_NAME') and settings.BLOB_STORAGE_ACCOUNT_NAME != "":
-            service = service_class(
-                settings.BLOB_STORAGE_ACCOUNT_NAME,
-                settings.BLOB_STORAGE_ACCOUNT_KEY,
-                protocol=settings.PROTOCOL,
-            )
-        else:
-            raise SkipTest('BLOB_CONNECTION_STRING or BLOB_STORAGE_ACCOUNT_NAME must be populated to run this test')
-
-        self._set_test_proxy(service, settings)
-        return service
-
-    def _create_premium_storage_service(self, service_class, settings):
-        if hasattr(settings, 'PREMIUM_CONNECTION_STRING') and settings.PREMIUM_CONNECTION_STRING != "":
-            service = service_class(connection_string=settings.PREMIUM_CONNECTION_STRING)
-        elif settings.IS_EMULATED:
-            service = service_class(is_emulated=True)
-        elif hasattr(settings, 'PREMIUM_STORAGE_ACCOUNT_NAME') and settings.PREMIUM_STORAGE_ACCOUNT_NAME != "":
-            service = service_class(
-                settings.PREMIUM_STORAGE_ACCOUNT_NAME,
-                settings.PREMIUM_STORAGE_ACCOUNT_KEY,
-                protocol=settings.PROTOCOL,
-            )
-        else:
-            raise SkipTest('PREMIUM_CONNECTION_STRING or PREMIUM_STORAGE_ACCOUNT_NAME must be populated to run this test')
-
-        self._set_test_proxy(service, settings)
-        return service
-
-    def _create_remote_storage_service(self, service_class, settings):
-        if settings.REMOTE_STORAGE_ACCOUNT_NAME and settings.REMOTE_STORAGE_ACCOUNT_KEY:
-            service = service_class(
-                settings.REMOTE_STORAGE_ACCOUNT_NAME,
-                settings.REMOTE_STORAGE_ACCOUNT_KEY,
-                protocol=settings.PROTOCOL,
-            )
-        else:
-            print("REMOTE_STORAGE_ACCOUNT_NAME and REMOTE_STORAGE_ACCOUNT_KEY not set in test settings file.")
-        self._set_test_proxy(service, settings)
-        return service
-
     def assertNamedItemInContainer(self, container, item_name, msg=None):
```

Comment:
Now that we have the chance - this function name could be refactored to the python snake casing